public_html/index.html

```diff
@@ -217,7 +242,7 @@ <h3>Testimonials</h3>
                                 regulatory situations. I have worked
                                 closely with her, and she brings tenacious advocacy and deep industry experience to
                                 achieve practical solutions.”</p>
-                            <p>&mdash; A state government lawyer.</span>
+                            <p class="testimonial__author">&mdash; A state government lawyer.</p>
```

Comment:
This class, `testimonial__author", should be right-aligned and *not* underlined.

Reply:
:Not underlined" is working